arviz/data/io_pyro.py

```diff
@@ -1,8 +1,7 @@
 """pyro-specific conversion code."""
-import numpy as np
-
 from .inference_data import InferenceData
 from .base import dict_to_dataset
+from ..utils import expand_dims
```

Comment:
Gonna be super picky here, it would be great if all imports were all the same

For example in this case its 
`..utils import expand_dims`
but in other modules its

```
from .. import utils
utils.expand_dims
```
I personally prefer the second since it makes it clear the method is defined somewhere else, but consistency matters more than that